From 381f68199cdfa530f284d58dbbd012b7de126781 Mon Sep 17 00:00:00 2001
From: Martino Facchin <m.facchin@arduino.cc>
Date: Mon, 9 Mar 2020 18:47:11 +0100
Subject: [PATCH 16/31] HACK: remove 3status bytes macronix quirk

This is needed by the flash memory we chose for Portenta.
Need to discuss with mbed devs to understand how to keep the quirk "alive"
---
 .../storage/blockdevice/COMPONENT_QSPIF/QSPIFBlockDevice.cpp     | 1 -
 1 file changed, 1 deletion(-)

diff --git a/components/storage/blockdevice/COMPONENT_QSPIF/QSPIFBlockDevice.cpp b/components/storage/blockdevice/COMPONENT_QSPIF/QSPIFBlockDevice.cpp
index 33f716b777..904b9f7bfb 100644
--- a/components/storage/blockdevice/COMPONENT_QSPIF/QSPIFBlockDevice.cpp
+++ b/components/storage/blockdevice/COMPONENT_QSPIF/QSPIFBlockDevice.cpp
@@ -1059,7 +1059,6 @@ int QSPIFBlockDevice::_handle_vendor_quirks()
             // 3. Should never attempt to enable 4-byte addressing (it causes reads and writes to fail)
             tr_debug("Applying quirks for macronix");
             _needs_fast_mode = true;
-            _num_status_registers = 3;
             _read_status_reg_2_inst = QSPIF_INST_RDCR;
             _attempt_4_byte_addressing = false;
             break;
-- 
2.27.0

